### ISSUE TITLE
MissingPluginException

### ISSUE THREAD

**Issue by UnifySoftTech** [g123k/flutter_torch_compat#7]
ERROR:flutter/lib/ui/ui_dart_state.cc(157)] Unhandled Exception: MissingPluginException(No implementation found for method turnOff on channel g123k/torch_compat)

This is what i get when i try to use this package

**Comment by g123k** [comment]
@UnifySoftTech Do the other methods work?

**Comment by nathanielheitsch** [comment]
Same issue here:
`MissingPluginException (MissingPluginException(No implementation found for method turnOn on channel g123k/torch_compat))`

Awaiting `TorchCompat.hasTorch` works fine however cannot test the `turnOff` method obviously.

Building on Samsung Galaxy S9+ (Android 10)

**Comment by UnifySoftTech** [comment]
I could debug that it was happening due to camera view being open seems
like some sort of incompatible with each other, If you try to access it
somewhere where camera is not open then it works fine.

Thanks

**Comment by g123k** [comment]
Only one instance of the camera can be opened at once.
So this plugin can only be used if no camera is used elsewhere.

**Comment by UnifySoftTech** [comment]
Yes. Wish flutter would include this into their official package

**Comment by dayron9110** [comment]
Any solution for this?

**Comment by UnifySoftTech** [comment]
Not yet

**Comment by qianxuechao** [comment]
http://blog.z7ge.com/archives/flutter-missingpluginexception-bu-yao-huang 希望能帮到你
